scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Not nice to over-write a user specified parameter: MOT_SAVED_PATH. Maybe rename MOT_SAVED_PATH to MOT_ROOT_PATH, and then MOT_SAVED_PATH becomes mot_path</li><li>Needs some explanation why we evaluate on the "train" subfolder, and that for the test set one has to use the evaluation server. I assume there is no validation dataset?</li><li>Format this (and all/most other cells in this notebook) using black. There are multiple libraries, e.g. https://github.com/drillan/jupyter-black</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>